src/table/headerRenderers/dataHeader.component.test.tsx

```diff
@@ -138,4 +142,22 @@ describe('Data Header', () => {
       expect(onSort).toHaveBeenCalledWith('test', 'desc');
     });
   });
+
+  it('displays tooltip when user hovers over column name', async () => {
+    createView();
+    const header = screen.getByText('Test');
```

Comment:
I'm personally a fan of being more specific about the type of element we're trying to access. The `Tooltip` is wrapped around the header sort label, in which case I would put `screen.getByTestId('sort test')`. This allows you to be more descriptive about what part you're trying to test, compared to just referring to a piece of text that could appear anywhere on the screen

Reply:
I'm trying to emulate hovering over the header name specifically, but I suppose we can target the overall sort label as well

Reply:
It might help if I changed the header name to be more specific so that the label is unique sounding when I target it

Reply:
Ah, because the tooltip is wrapped around the sort label, which is wrapped around the header name, so hovering over the sort icon itself also causes the tooltip to appear. So I guess to hover over the header name itself, we'd have to put the tooltip solely around the header name. Is it worth doing? Unsure. The test does pass anyway